Makefile

```diff
@@ -15,37 +15,53 @@
 
 
 ##
-## -- commands and flags
-##    ------------------
-GCC       = gcc -c
-CFLAGS   += -I inc/
-CFLAGS   += -I /usr/include/mysql-cppconn-8/
-
-AR		  = ar -cr
-
-
-##
-## -- Some macros to make our job easier
+## -- Some macros to help with the build
 ##    ----------------------------------
-LIB=libcba.a
-SRC=$(sort $(subst src/,,$(wildcard src/*.cc)))
-OBJ=$(subst .cc,.o,$(SRC))
+TGT = libcba.a
+SRC = $(sort $(subst src/,,$(wildcard src/*.cc)))
+OBJ = $(subst .cc,.o,$(SRC))
+TST = $(sort $(subst tst/,,$(wildcard tst/*.cc)))
+EXE = $(subst .cc,,$(TST))
 
+CFLAGS += -I inc/
+CFLAGS += -I /usr/include/mysql-cppconn-8/
+CFLAGS += -std=c++14
 
+CC = gcc -c
+AR = ar rcs
 
-.PHONY: all
-all: lib/$(LIB) bin/test
 
-bin/test: obj/$(OBJ)
-	mkdir -p bin
-	gcc -o $@ $< -lmysqlcppconn8 -lstdc++
+##
+## -- This is the main target
+##    -----------------------
+.phony: all
+all: lib/$(TGT) bin/$(EXE)
```

Comment:
Add a target that executes the tests

Reply:
This has been addressed.  `make test` will now run the tests, but `make` will build everything.